src/components/calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../App.css';
+
```

Comment:
This is the file that you have to capitalize the name.

Reply:
Thank you for the suggested changes.